designer/server/src/main/scala/pl/touk/nussknacker/ui/api/AkkaToTapirStreamExtension.scala

```diff
@@ -0,0 +1,53 @@
+package pl.touk.nussknacker.ui.api
+
+import akka.stream.Materializer
+import akka.stream.scaladsl.{Source, StreamConverters}
+import akka.util.ByteString
+import sttp.capabilities.Streams
+import sttp.capabilities.akka.AkkaStreams
+import sttp.tapir.CodecFormat.OctetStream
+import sttp.tapir.EndpointIO.StreamBodyWrapper
+import sttp.tapir.{Codec, CodecFormat, DecodeResult, EndpointIO, Schema, StreamBodyIO}
+
+import java.io.InputStream
+import java.util.concurrent.TimeUnit
+import scala.concurrent.duration.FiniteDuration
+
+class AkkaToTapirStreamExtension(implicit val mat: Materializer) {
```

Comment:
shouldn't we have some well-named trait for this? and use the trait everywhere but the place when we construct services?